spec/system/support_interface/audit_trail_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'rails_helper'
+
+RSpec.feature 'See applications' do
+  around do |example|
+    Timecop.freeze(Time.zone.local(2019, 10, 1, 12, 0, 0)) do
+      example.run
+    end
+  end
+
+  scenario 'Support user visits application audit page' do
+    given_i_am_a_support_user
+    and_there_is_an_application_in_the_system_logged_by_a_candidate
+    and_a_vendor_updates_the_application_status
+    and_i_visit_the_support_page
+
+    when_i_click_on_an_application_history
+    then_i_should_be_on_the_application_history_page
+    then_i_should_be_able_to_see_history_events
+  end
+
+  def given_i_am_a_support_user
+    page.driver.browser.authorize('test', 'test')
+  end
+
+  def and_there_is_an_application_in_the_system_logged_by_a_candidate
+    candidate = create :candidate, email_address: 'alice@example.com'
```

Comment:
I would make this dynamic, as I'm getting 'Validation failed: Email address Email address is already in use' in this and another spec. Which is weird

Reply:
> Regarding dates, there's some style guidance here: https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#dates (10 October 2019 rather than October 10, 2019)
> 
> And times: https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#times
> 
> So maybe something like:
> `29 October 2019 at 5:30pm` (seconds don't matter in UI, could keep them there as a datetime attribute on a `time` element)

thanks again @fofr. I've changed the date format - I used the abbreviated months to avoid the line wrap (the guidance suggests this if space is tight). I'm not sure what to do with the times, because this is a support interface the seconds could well be relevant, e.g. if you are trying to work out whether a 20 second timeout is being applied. What do you think? Is `29 Oct 2019 17:30:55` acceptable or do we really need to drop the seconds?

Reply:
Thanks @mnacos - I just noticed a build failure with the same error. Need to work out what's going on there with cleanup etc.

Reply:
I've deleted and recreated my test database and the error has stopped. I did notice the fake data generator failure, too, my suggestions above seem to fix it.